components/raftstore-v2/src/operation/query/mod.rs

```diff
@@ -2,34 +2,112 @@
 
 //! There are two types of Query: KV read and status query.
 //!
-//! KV Read is implemented in local module and lease module (not implemented
-//! yet). Read will be executed in callee thread if in lease, which is
+//! KV Read is implemented in local module and query module.
+//! Read will be executed in callee thread if in lease, which is
 //! implemented in local module. If lease is expired, it will extend the lease
 //! first. Lease maintainance is implemented in lease module.
 //!
 //! Status query is implemented in the root module directly.
+//! Follower's read index and replica read is implemenented replica.rs.
```

Comment:
Inconsistent comment. It's called xxx module instead of xxx.rs in the above sentences.